eventtracking/tasks.py

```diff
@@ -4,7 +4,7 @@
 import json
 
 from celery.utils.log import get_task_logger
-from celery import task
```

Comment:
Its removed in celery5.0 so updating it.  Its compatible with 4.4.7 and 5.0 also.